### ISSUE TITLE
less-watch-compiler.config.json not being loaded

### ISSUE THREAD

**Issue by kevinjyee** [jonycheung/deadsimple-less-watch-compiler#128]
**Which version are you using (less-watch-compiler --version to find out)?**
1.15.0

**Is the issue reproducible after updating to the latest version ( npm update less-watch-compiler)?**
Yes.

**Describe the bug**
`less-watch-compiler.config.json` is not being loaded in after bumping from version `1.14.6` to `1.15.0`

**To Reproduce**
1. Folder setup
  ─ src
      └── styles
          └─  less
               └─   styles.less
  ─ less-watch-compiler.config.json

and my `less-watch-compiler.config.json` file looks look this
 ```
{
    "watchFolder": "src/styles/less",
    "outputFolder": "src/styles/less",
    "sourceMap": false,
    "runOnce": false,
    "enableJs": true,
    "mainFile": "styles.less"
}
```

1. Run `less-watch-compiler` 

**Expected behavior**
Config file `less-watch-compiler.config.json` to be loaded in to where I don't have to manually specify `watchFolder` and `outputFolder` in the `args`

**Actual Behavior**
Error thrown
`Missing arguments. Example: node less-watch-compiler.js FOLDER_TO_WATCH FOLDER_TO_OUTPUT`

**Comment by jonycheung** [comment]
I found a solution. I'll submit a PR soon

**Comment by jonycheung** [comment]
For now, you should be able to add `--config less-watch-compiler.config.json` to get around it.

**Comment by jonycheung** [comment]
This was released in `v1.15.1` Thanks.
